db/migrate/20160324174132_create_deployments.rb

```diff
@@ -0,0 +1,21 @@
+class CreateDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string   :deployment_type
```

Comment:
@alongoldboim is `deployment_type` "origin", "enterprise", "atomic", etc.? let's also add `deployment_version` (e.g. `3.2` for the final release, and `3.1` for current development).

@detiber is this `variant_version` in your yaml file or is it not covered yet?